reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
I agree that it'd be a pain with the current build/dev system to inject that properly into the main stylesheet.  I also agree that putting it into the reddit-public repo would be non-ideal both for cleanliness and future development surprises.

Crazy idea: just duplicate those style rules in a `<style>` tag on the front page. Then you can just put it into the template in this repo.  It's a pretty small number of bytes and by the same logic of not inlining the SVG file it's not worth it to bloat up the main stylesheet for something that will only be on occasionally.

I'm not really convinced that those rules really need to be shared anyway -- it's not clear to me that a change to the status bar on live would mean we should change the thing on the frontpage.  They're two different UI elements that happen to look the same right now.  Chances are that a dev changing the status bar wouldn't think to check the front page UI anyway.